deploy/helm/sumologic/conf/metrics/metrics.conf

```diff
@@ -19,6 +19,15 @@
     </exclude>
   </filter>
 {{- end }}
+{{- if eq .Values.sumologic.collectionMonitoring "false" }}
+  <filter prometheus.metrics**> 
+    @type grep
+    <exclude>
+      key @metric
+      pattern /^up|^prometheus_remote_storage_.*|^fluentd_.*|^fluentbit.*|^otelcol.*$/
```

Comment:
Ideally we would be able to remove from the Prometheus side so we're not sending unnecessary metrics to Fluentd, though I understand we can't template in the `values.yaml` file so this would be difficult to achieve...

Reply:
yes, that's why I added the filter here